share/functions/__fish_npm_helper.fish

```diff
@@ -38,7 +38,7 @@ function __npm_filtered_list_packages
             test -f $path || continue
             # Using `string replace` directly is slow because it doesn't know to stop looking after a match cannot be
             # found in the alphabetically sorted list. Use `look` for its binary search support.
-            look '  "'(commandline -ct) $path | string replace -rf '^  "(.*?)".*' '$1'
+            look '  "'(commandline -ct) $path | string replace -rf '^  "(.*?)".*' '$1' | string unescape --style=regex
```

Comment:
I'm not sure what's going on here - this looks like it was something hyper-specific to npm completions? We're already using `look` so maybe we can optimize this completion specifically without messing with builtins.